drivers/tty/serdev/pibridge_trace.h

```diff
@@ -302,6 +495,12 @@ DECLARE_EVENT_CLASS(pibridge_kfifo_data_class,
 	)
 );
 
+/*
+ * pibridge_discard_data
+ *
+ * Info: the bytes contained in the receive fifo being discarded.
+ * Time: after the header of an IO response packet has been received.
```

Comment:
This looks like a copy/paste error. How about:
"after a larger than expected packet has been received or a timeout occurred"